pkg/cmd/runsDownload.go

```diff
@@ -80,7 +81,9 @@ func executeRunsDownload(cmd *cobra.Command, args []string) {
 	var console = utils.NewRealConsole()
 
 	apiServerUrl := bootstrapData.ApiServerURL
-	log.Printf("The API sever is at '%s'\n", apiServerUrl)
+	log.Printf("The API server is at '%s'\n", apiServerUrl)
+
+	apiClient := auth.GetAuthenticatedAPIClient(apiServerUrl, fileSystem, galasaHome)
```

Comment:
Can't this fail ? If it does, then this command needs to fail ?

Reply:
If the login attempt fails, then it will continue with an API client that does not use a bearer token for now. Once we turn off the functionality to continue without a token, then the command can fail and will return an error.